### PR TITLE
Add disclaimer

### LINKED ISSUE #332: Add info button with disclaimer/acknowledgement text
[METOC webviewer disclaimer.docx](https://github.com/Deltares/fews-web-oc/files/12925402/METOC.webviewer.disclaimer.docx)

### PR DESCRIPTION
**Pull request by hvangeffen**
Closes #332 

[METOC.webviewer.disclaimer.docx](https://github.com/Deltares/fews-web-oc/files/13040612/METOC.webviewer.disclaimer.docx)

Converted with pandoc to html

### CODE REVIEW

File: src/layouts/Default.vue [hunk 1]
```diff
@@ -5,6 +5,7 @@
       <v-spacer />
       <div>{{ webOcTitle }}</div>
       <v-spacer />
+      <DisclaimerPopup />
```
Comment (vanmeeuwen) [inline src/layouts/Default.vue:8]: We probably need to fix the Z-index to be in front of at least the search locations button and the sidebar

Comment (hvangeffen) [reply src/layouts/Default.vue:8]: This is done in the SplashScreen pull request

File: src/components/DisclaimerPopup.vue [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+          <li>
+            <p>7.1 Any and all of User's personal data that User may provide to
+              Deltares for purposes of his/her use of the Viewer, will be processed in
+              accordance with the General Data Protection Regulation (EU) 2016/679 and
+              the Netherlands GDPR implementation Law 'Uitvoeringswet Algemene
+              verordering gegevensbescherming' (stb 2018, 145). Our Privacy Statement
+              sets out when and why we collect the data, exactly what that data is,
+              and how it will be processed and, above all, protected. Please read the
+              statement carefully, available on this website:
+              <a href="https://wetten.overheid.nl/BWBR0040940/2021-07-01">https://wetten.overheid.nl/BWBR0040940/2021-07-01</a>.</p>
```
Comment (vanmeeuwen) [inline src/components/DisclaimerPopup.vue:235]: Remove hyperlink (based on advice of Riemer who mentions that links to dutch pages are not really useful)

Comment (hvangeffen) [reply src/components/DisclaimerPopup.vue:235]: Added that one manually as it wasnt a link in the docx
Will change it back

**Review by vanmeeuwen — APPROVED** [review]

**Pull request merged**